### PR TITLE
Travis echidna

### PR DESCRIPTION
**Pull request by gkellogg**
* Add common files from json-ld-wg.
* Add files for checking examples.
* Markup examples so that examples can be checked automatically.
* Add rake task to rebuild reports/manifests.nt.
* Add Travis configuration.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-streaming/pull/14.html" title="Last updated on Apr 18, 2020, 8:18 PM UTC (c02b157)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-streaming/14/cf975e4...c02b157.html" title="Last updated on Apr 18, 2020, 8:18 PM UTC (c02b157)">Diff</a>

### CODE REVIEW

**Comment by gkellogg** [comment]
This will be an update when the tokens for this version of ECHIDNA are available, as well as adding the ECHIDNA configuration file.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      language: javascript
+      # Only on push to "publications" branch
+      if: branch = publications AND type = push
+      env:
+        - URL="https://w3c.github.io/json-ld-streaming/ECHIDNA"
+        - DECISION="https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2018/2018-09-28-json-ld#resolution2"
+        - secure: "lLdok36PZzMuoXtVGDLtqEPXscvbM/lAg3LUZ9rrFysqflIGZYbA4A19u7J9abB9Z5OLyOlUGbIoTxENbeO8A0lEsEjwYqh+Hg0FL2ZLMR8xHmjfQ+wiO/q8RURogaun1FZXIm1chaYarf2V2QBmyxpzX3+Hqhaw4oji16NrPh+tcdZVRoER24kV6AeDowuS9ZXviZP3fqQhRYG114lUY6H94GdqYWGvVrRjA6z8bpyHB+yDD+8n7ma0MNiufbhBzeq1uz57TJebsr8gSs8rp7VH8w3NI3Gf6Ig5XdhOGcFssZNm5YWY/KaS5aihYoAvMJguKQ5ALARvJzpzmdMBz9KsBRKBgtPW56r2XD1ITlFpo3qVRxLsyGTu8y3hWY6U8SqMdpdYF1dwHLfoL6vO5ATn/CZ3B4POmSW5FepwOcOAluLE6LwBgd35h8Fypt4g5ruceZyFddIcvAYlUBPZPIi/tItQVz6zz+xb8aL7vR2/ekLKFTckbskb/gN60fsIjUumxQeOslw9mGO6jUch67eizzaL2Wob6VKDrxoymb873LbepTgOFFksaUp8lwbtsUPgOtJQ7hWCH2StZmIioh4LMumcRKm0EcIdtn5vh/f1MDETVmLpgKt0HeMkaH6bUCe6km3ZedaRfObLqEV3+tVyo4T4f+u0eo2S/JBFGOU="
+      script:
+        - curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN" --data "editorial=true"
```
Comment (iherman) [inline .travis.yml:44]: As far as I remember, the "editorial=true" was relevant for CR only. Not for a WG Note

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        - docker run -it stupchiy/checklink -b -X '(http|https|mailto):' "https://labs.w3.org/spec-generator/?type=respec&url=https://rawgit.com/w3c/json-ld-streaming/${TRAVIS_BRANCH}/index.html?specStatus=WD&shortName=json-ld11-streaming"
+        - bundle exec rake check_html
+    - stage: echidna
+      name: "Echidna Publication"
+      language: javascript
+      # Only on push to "publications" branch
+      if: branch = publications AND type = push
+      env:
+        - URL="https://w3c.github.io/json-ld-streaming/ECHIDNA"
+        - DECISION="https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2018/2018-09-28-json-ld#resolution2"
```
Comment (gkellogg) [inline .travis.yml:41]:
```suggestion
        - DECISION="https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2020/2020-04-17-json-ld#resolution4"
```

File: .travis.yml (hunk 1 above)
Comment (gkellogg) [reply .travis.yml:44]:
```suggestion
        - curl "https://labs.w3.org/echidna/api/request" --data "url=$URL" --data "decision=$DECISION" --data "token=$TOKEN"
```

**Pull request merged**